<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset theme="xware-desktop"/>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_loginAndStart">
      <attribute name="title">
       <string>启动与登录</string>
      </attribute>
      <widget class="QGroupBox" name="group_etmStartWhen">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>10</y>
         <width>280</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>ETM何时启动</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="radio_backendStartWhen1">
           <property name="toolTip">
            <string>Xware开机后即上线开始下载。适用于长期挂机的用户。</string>
           </property>
           <property name="text">
            <string>随xwared启动</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_backendStartWhen2">
           <property name="toolTip">
            <string>后端开启与否由前端控制。做出的选项重启电脑后依然保持。</string>
           </property>
           <property name="text">
            <string>启动状态维持手动设定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_backendStartWhen3">
           <property name="toolTip">
            <string>这是Windows上的迅雷的标准行为。图形界面关闭后停止下载。</string>
           </property>
           <property name="text">
            <string>随前端启动而启动，随前端退出而退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="group_frontend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>280</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>前端启动登录</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>33</y>
          <width>218</width>
          <height>89</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_loginUsername">
           <property name="text">
            <string>用户名</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_loginUsername</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_loginUsername"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_loginPassword">
           <property name="text">
            <string>密码</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_loginPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_loginPassword">
           <property name="inputMask">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBox_autoLogin">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>自动登录前端</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_autoStartFrontend">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>开机启动前端</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="group_initManaged">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>280</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>xwared托管</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radio_managedBySystemd">
           <property name="toolTip">
            <string>systemd 用户会话</string>
           </property>
           <property name="text">
            <string>由用户态systemd托管</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_managedByUpstart">
           <property name="text">
            <string>由用户态upstart托管</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_managedByAutostart">
           <property name="text">
            <string>简单的自动启动(不推荐)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_managedByNothing">
           <property name="text">
            <string>不托管(不推荐)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_frontend">
      <attribute name="title">
       <string>前端</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_enableDevelopersTools">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>251</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>开启后，可以在页面右键点击，打开开发者工具。</string>
       </property>
       <property name="text">
        <string>在网页中启用 Webkit 开发者工具</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>12</y>
         <width>231</width>
         <height>170</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_allowFlash">
          <property name="toolTip">
           <string>一些功能，比如复制URL，需要Flash支持。</string>
          </property>
          <property name="text">
           <string>允许页面中使用Adobe Flash等插件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_minimizeToSystray">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>最小化到托盘</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_closeToMinimize">
          <property name="text">
           <string>关闭窗口时最小化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_showMonitorWindow">
          <property name="text">
           <string>显示悬浮窗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_popNotifications">
          <property name="text">
           <string>弹出通知窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_notifyBySound">
          <property name="text">
           <string>播放通知声音</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>661</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_watchClipboard">
          <property name="text">
           <string>监视剪切板</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plaintext_watchPattern"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_monitorFullSpeed">
          <property name="text">
           <string>悬浮窗满速</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_monitorFullSpeed</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_monitorFullSpeed">
          <property name="suffix">
           <string> KiB/s</string>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_mount">
      <attribute name="title">
       <string>挂载</string>
      </attribute>
      <widget class="QTableWidget" name="table_mounts">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>891</width>
         <height>241</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Windows路径</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Unix路径</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>状态</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>261</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_addMount">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_removeMount">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_backend">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>后端</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_maxRunningTasksNum">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_maxRunningTasksNum">
          <property name="text">
           <string>最多可同时下载任务数</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_maxRunningTasksNum</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_uSpeedLimit">
          <property name="text">
           <string>上传限速</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_uSpeedLimit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_uSpeedLimit">
          <property name="toolTip">
           <string>无限制设置为-1</string>
          </property>
          <property name="suffix">
           <string> KiB/s</string>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_dSpeedLimit">
          <property name="text">
           <string>下载限速</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_dSpeedLimit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_dSpeedLimit">
          <property name="toolTip">
           <string>无限制设置为-1</string>
          </property>
          <property name="suffix">
           <string> KiB/s</string>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_loginUsername</tabstop>
  <tabstop>lineEdit_loginPassword</tabstop>
  <tabstop>checkBox_autoLogin</tabstop>
  <tabstop>checkBox_autoStartFrontend</tabstop>
  <tabstop>radio_managedBySystemd</tabstop>
  <tabstop>radio_managedByUpstart</tabstop>
  <tabstop>radio_managedByAutostart</tabstop>
  <tabstop>radio_managedByNothing</tabstop>
  <tabstop>radio_backendStartWhen1</tabstop>
  <tabstop>radio_backendStartWhen2</tabstop>
  <tabstop>radio_backendStartWhen3</tabstop>
  <tabstop>checkBox_allowFlash</tabstop>
  <tabstop>checkBox_minimizeToSystray</tabstop>
  <tabstop>checkBox_closeToMinimize</tabstop>
  <tabstop>checkBox_showMonitorWindow</tabstop>
  <tabstop>checkBox_popNotifications</tabstop>
  <tabstop>checkBox_notifyBySound</tabstop>
  <tabstop>spinBox_monitorFullSpeed</tabstop>
  <tabstop>checkBox_enableDevelopersTools</tabstop>
  <tabstop>checkBox_watchClipboard</tabstop>
  <tabstop>plaintext_watchPattern</tabstop>
  <tabstop>table_mounts</tabstop>
  <tabstop>btn_addMount</tabstop>
  <tabstop>btn_removeMount</tabstop>
  <tabstop>spinBox_maxRunningTasksNum</tabstop>
  <tabstop>spinBox_dSpeedLimit</tabstop>
  <tabstop>spinBox_uSpeedLimit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="rc/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
